scenarios/23_Trapped.cfg

```diff
@@ -1,47 +1,74 @@
 #textdomain wesnoth-A_New_Order
 
 #define DUNEFOLK_TALK
+    {VARIABLE dunefolk_price {ON_DIFFICULTY 200 225 250}}
```

Comment:
I chose it every time... also since I made it so the gold gets paid to their side, lowering the price you pay could actually make it harder since then they won't be able to help you as much

Reply:
> You can edit the gold where you give them bonus gold on top of the price you pay them then.

OK I did that in 4bdbac0